syntaxes/test/data/inline-template.ts.snap

```diff
@@ -93,4 +95,26 @@
 #^^^ inline-template.ng
 >export class TMComponent{}
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^ inline-template.ng
+>
+>/* Template syntax tests */
+#^^^^^^^^^^^^^^^^^^^^^^^^^^^^ inline-template.ng
```

Comment:
Could you please explain why regular TypeScript nodes are also labelled `inline-template.ng`?

Reply:
This seems to be a quirk of the testing framework inherited from TextMate. It require a base scope for the grammar language (in this case, `inline-template.ng`), which becomes the base grammar name of everything the grammar is applied on. More specific grammar names are stacked as the grammar proceeds.

Reply:
ah, I see. We might want to document that in a README.

Reply:
I will add it to the documentation in a next PR.